Initium-Odp/war/odp/javascript/script.js

```diff
@@ -759,6 +759,14 @@ function createNewGroup(eventObject)
 }
 
 
+function deleteGroup(eventObject)
+{
+	confirmPopup("Confirmation", "Are you sure you want to delete your group?\n\nThis cannot be undone", function(){
+		doCommand(eventObject, "GroupDelete")
```

Comment:
Maybe it would be good to close the page popup in a callback? If you do, make sure to check for errors first.

IIRC, closePagePopup() or something similar